pkg/controller/controller_test.go

```diff
@@ -569,21 +580,70 @@ func TestGetSecretReference(t *testing.T) {
 			expectRef:    nil,
 			expectErr:    true,
 		},
+		"simple - PVC name annotations not supported for Provision and Delete": {
+			secretParams: provisionerSecretParams,
+			params: map[string]string{
+				prefixedProvisionerSecretNameKey: "static-${pv.name}-${pvc.namespace}-${pvc.name}-${pvc.annotations['akey']}",
+			},
+			pvc:       &v1.PersistentVolumeClaim{},
```

Comment:
Can you fill in name and namespace for this pvc too, so we know it's not failing because of name or namespace missing?  Also does pv name need to be passed in?